### PR TITLE
Fix Media Element Transport controls visibility

### LINKED ISSUE #1745: [BUG] Full Screen button on Windows does not adhere to `ShouldShowPlaybackControls`
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Did you read the "Reporting a bug" section on Contributing file?

- [X] I have read the "Reporting a bug" section on Contributing file: https://github.com/CommunityToolkit/Maui/blob/main/CONTRIBUTING.md#reporting-a-bug

### Current Behavior

When setting `MediaElement.ShouldShowPlaybackControls` to false all on-screen platform playback controls are not shown, however, the full screen button still shows up on Windows.

### Expected Behavior

When setting `MediaElement.ShouldShowPlaybackControls` to false also the full screen button is not shown.

### Steps To Reproduce

1. Start the sample app, go to the MediaElement page on Windows
2. Toggle Show Playback Controls switch to be false
3. Interact with the MediaElement and observe how the full screen button is still shown

### Link to public reproduction project repository

https://github.com/CommunityToolkit/Maui

### Environment

```markdown
- .NET MAUI CommunityToolkit: (MediaElement package) 3.1.0
- OS: Not important
- .NET MAUI: 8.0.7
```

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by ne0rrmatrix**
### Description of Change ###
1. In the `MauiMediaElement` constructor, we added setting of the `Visibility`. Now, the `Visibility` property is set to `Collapsed` in the constructor.

2. A new conditional statement has been added to the `OnMediaPlayerElementPointerMoved` method. If the `Visibility` property of `mediaPlayerElement.TransportControls` is `Collapsed`, the `Visibility` property of `buttonContainer` is also set to `Collapsed` and the method returns early. This behavior was not present in the previous version of the method.

### Video of Bug

https://github.com/CommunityToolkit/Maui/assets/4167863/cd2226fa-2d1e-4c7e-8e24-db59f039698f

### Video with Fix applied

https://github.com/CommunityToolkit/Maui/assets/4167863/296a27ec-3e9b-4078-bfcc-f95b03c39118

 ### Linked Issues ###

 - Fixes #1745

 ### PR Checklist ###
 - [x] Has a linked Issue, and the Issue has been `approved`(bug) or `Championed` (feature/proposal)
 - [ ] Has tests (if omitted, state reason in description)
 - [ ] Has samples (if omitted, state reason in description)
 - [x] Rebased on top of `main` at time of PR
 - [x] Changes adhere to [coding standard](https://github.com/CommunityToolkit/Maui/blob/main/CONTRIBUTING.md#contributing-code---best-practices)
 - [ ] Documentation created or updated: https://github.com/MicrosoftDocs/CommunityToolkit/pulls 

 ### Additional information ###
This applies to a windows specific issue with a button that did not comply with the transport control settings. This PR fixes that and the button is now compliant. If the transport controls are turned off while the controls are visible the full screen controls will stay on until it time out but will not reappear until the controls are turned back on.

### CODE REVIEW

File: src/CommunityToolkit.Maui.MediaElement/Views/MauiMediaElement.windows.cs [hunk 1]
```diff
@@ -64,7 +64,8 @@ public MauiMediaElement(MediaPlayerElement mediaPlayerElement)
 		{
 			HorizontalAlignment = Microsoft.UI.Xaml.HorizontalAlignment.Right,
 			VerticalAlignment = Microsoft.UI.Xaml.VerticalAlignment.Top,
-			Width = 40,
+			Visibility = mediaPlayerElement.TransportControls.Visibility,
+		Width = 40,
```
Comment (jfversluis) [inline src/CommunityToolkit.Maui.MediaElement/Views/MauiMediaElement.windows.cs:68]:
```suggestion
			Visibility = mediaPlayerElement.TransportControls.Visibility,
			Width = 40,
```

Comment (jfversluis) [reply src/CommunityToolkit.Maui.MediaElement/Views/MauiMediaElement.windows.cs:68]: Fix weird indentation

**Review by vhugogarcia — APPROVED** [review]
Looks good to me! Thanks @ne0rrmatrix

**Comment by nonoc** [comment]
Hi guys, I'm using net9 preview 2 and all updates from Maui controls, etc, I'm trying to watch locally in a Mac using the Media Element but I have not any luck, any help please.
Thanks

**Comment by bijington** [comment]
> Hi guys, I'm using net9 preview 2 and all updates from Maui controls, etc, I'm trying to watch locally in a Mac using the Media Element but I have not any luck, any help please. That was happening with 3.1.0 so I have had to turn down to 3.0.1
> 
> It was happening too in Net8, for this reason I'll try with Net 9 but without luck.
> 
> Thanks

Can you please open an issue detailing your problem and the steps that you have followed? I will add that I doubt we will invest much time in .NET 9 right now given it is still in preview

**Comment by nonoc** [comment]
Hi I said that it was happening on Net 8 too, but I don't think that could depend on the framework, I explain that my idea to up to Net 9 was because maybe it correct the issue, 
Basically it is not playing the media (mp4) on local system files on Mac, I have tried filesystem:// and all document on the MediaElement but I didn't any luck. that is working with URI, I could think that it could be for SandBox security on Mac, so I can't watch any Video outside of the App folder then?
Thank you very much.

**Comment by bijington** [comment]
Without seeing your code it's difficult to say. Please open a new issue and explain your scenario and we can go from there

**Pull request merged**